aries-core/src/js/components/helpers/AppIdentity/AppIdentity.js

```diff
@@ -0,0 +1,53 @@
+import React, { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Button, Text } from 'grommet';
+import { Aruba, Hpe } from 'grommet-icons';
+
+const brandNames = {
+  hpe: 'HPE',
+  aruba: 'Aruba',
+};
+
+export const AppIdentity = forwardRef(
+  ({ brand, brandOnly, href, title, ...rest }, ref) => {
+    const textSize = 'small';
+
+    return (
+      <Button href={href} ref={ref} {...rest}>
+        <Box direction="row" align="center" gap="medium">
+          {brand === brandNames.hpe.toLowerCase() ? (
+            <Hpe color="brand" size="large" />
+          ) : (
+            undefined
+          )}
+          {brand === brandNames.aruba.toLowerCase() ? (
```

Comment:
I'm not loving the repetition here. Wondering if we can do the following?
```
const brands = {
  hpe: { name: 'HPE', logo: <Hpe color="brand" size="large" /> },
  aruba: { name: 'Aruba', logo: <Aruba color="orange!" /> },
};
```
then
```
{brand && brands[brand].logo}
```

Reply:
I like.